src/Symfony/Component/CssSelector/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+5.0.0
```

Comment:
Why not doing it on 4.4?

Reply:
Old habit of submitting features to master... I will rebase :)